pkg/pipeline/resource_pipeline_project_integration.go

```diff
@@ -158,24 +160,24 @@ func pipelineProjectIntegrationResource() *schema.Resource {
 	}
 
 	var unpackProjectIntegration = func(data *schema.ResourceData) (ProjectIntegration, error) {
-		d := &ResourceData{data}
+		d := &util.ResourceData{data}
 
 		projectIntegration := ProjectIntegration{
-			Name:                  d.getString("name"),
-			ProjectId:             d.getInt("project_id"),
-			MasterIntegrationId:   d.getInt("master_integration_id"),
-			MasterIntegrationName: d.getString("master_integration_name"),
-			Environments:          d.getList("environments"),
-			IsInternal:            d.getBool("is_internal"),
+			Name:                  d.GetString("name", false),
+			ProjectId:             d.GetInt("project_id", false),
+			MasterIntegrationId:   d.GetInt("master_integration_id", false),
+			MasterIntegrationName: d.GetString("master_integration_name", false),
+			Environments:          d.GetList("environments"),
+			IsInternal:            d.GetBool("is_internal", false),
 			Project:               unpackProject(d, "project"),
 			FormJSONValues:        unpackFormJSONValues(d, "form_json_values"),
 		}
 		return projectIntegration, nil
 	}
 
-	var packProjectIntegration = func(d *schema.ResourceData, projectIntegration ProjectIntegration) diag.Diagnostics {
+	var packProjectIntegration = func(ctx context.Context, d *schema.ResourceData, projectIntegration ProjectIntegration) diag.Diagnostics {
```

Comment:
Universal pack won't work?

Reply:
I'm not planning to update the code in this PR.